data/json/monster_special_attacks/spells.json

```diff
@@ -118,14 +118,15 @@
     "type": "SPELL",
     "id": "small_raptor_spawn",
     "name": { "str": "Spawn Raptors" },
-    "description": "Summons several zombiespawn raptors.",
+    "description": "Summons a few zombiespawn raptors.",
     "flags": [ "HOSTILE_SUMMON", "RANDOM_DAMAGE", "PERMANENT" ],
-    "valid_targets": [ "ground", "hostile" ],
-    "min_damage": 3,
-    "max_damage": 4,
-    "min_range": 3,
-    "max_range": 3,
-    "min_aoe": 2,
+    "valid_targets": [ "ground", "self" ],
+    "hit_self": true,
+    "min_damage": 1,
+    "max_damage": 2,
+    "min_range": 0,
+    "max_range": 1,
```

Comment:
```suggestion
```
this spell has no level, so 0 is the max range this spell will be. 0 is the default, so no reason to bloat the json